barberia/src/App/components/buttons.js

```diff
@@ -1,13 +1,35 @@
 import React from "react";
 import "./buttons.css";
 
+import { A } from "hookrouter";
 
-const hotShitButton = props => {
+
+
+const HotShitButton = props => {
   return (
     <div className="hot-shit-button" >
-        {props.children}
+      <A href={props.href}>
+        <div className="button-body">
+          {props.children}
```

Comment:
as we disciussed we split this into a title of `h3` and a description of `p` tag in the component